src/rai_bench/rai_bench/tool_calling_agent/predefined/manipulation_tasks.py

```
@@ -0,0 +1,97 @@
+
+######### VALIDATORS #########################################################################################
+move_to_point_ord_val_grab = OrderedCallsValidator(
+    subtasks=[move_to_point_subtask_grab]
+)
+move_to_point_ord_val_drop = OrderedCallsValidator(
+    subtasks=[move_to_point_subtask_drop]
+)
+
+
+def get_manipulation_tasks(
+    extra_tool_calls: List[int] = [0],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)